.drone.yml

```diff
@@ -40,66 +46,76 @@ trigger:
   event:
     - promote
 
+environment:
+  CHART_NAME: hocs-outbound-proxy
+  KUBE_CLUSTER: acp-notprod
+  KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+
 steps:
-    - name: checkout
-      image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
-      commands:
-        - git fetch --tags
-        - git checkout $${VERSION}
+  - name: checkout
+    image: alpine/git
+    commands:
+      - git fetch --tags
+      - git checkout ${VERSION}
+      - git submodule update --init --recursive
 
-    - name: prod deploy
-      image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
-      environment:
-        KUBE_CLUSTER: acp-prod
-        KUBE_NAMESPACE: ${DRONE_DEPLOY_TO}
-        KUBE_SERVER: https://kube-api-prod.prod.acp.homeoffice.gov.uk
-        KUBE_TOKEN:
-          from_secret: cs_prod
-      commands:
-        - ./kube/setup-kube.sh
-        - ./kube/helm-deploy.sh
-      depends_on:
-        - checkout
-      when:
-        target:
-          - '*-prod'
-          - '*-preprod'
+  - name: cs-qa
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    environment:
+      KUBE_NAMESPACE: cs-qa
+      KUBE_TOKEN:
+        from_secret: cs_notprod
+    commands:
+      - ./ci/helm/helm-deploy.sh --values ./helm/values-notprod.yaml
+    depends_on:
+      - checkout
+    when:
+      target:
+        - cs-qa
 
-    - &deploy
-      name: notprod deploy
-      image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
-      environment:
-        KUBE_CLUSTER: acp-notprod
-        KUBE_NAMESPACE: ${DRONE_DEPLOY_TO}
-        KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
-        KUBE_TOKEN:
-          from_secret: cs_notprod
-        VALUES_FILE: '--values ./helm/values-notprod.yaml'
-      commands:
-        - ./kube/setup-kube.sh
-        - ./kube/helm-deploy.sh
-      depends_on:
-        - checkout
-      when:
-        target:
-          exclude:
-            - '*-prod'
-            - '*-preprod'
+  - name: wcs-qa
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    environment:
+      KUBE_NAMESPACE: wcs-qa
+      KUBE_TOKEN:
+        from_secret: cs_notprod
+    commands:
+      - ./ci/helm/helm-deploy.sh --values ./helm/values-notprod.yaml
+    depends_on:
+      - cs-qa
+    when:
+      target:
+        - wcs-qa
+        - release
 
-    - <<: *deploy
-      name: cs-qa
-      environment:
-        KUBE_NAMESPACE: cs-qa
-      when:
-        target:
-          - release
+  - name: notprod
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    environment:
+      KUBE_NAMESPACE: ${DRONE_DEPLOY_TO}
+      KUBE_TOKEN:
+        from_secret: cs_notprod
+    commands:
+      - ./ci/helm/helm-deploy.sh --values ./helm/values-notprod.yaml
+    depends_on:
+      - checkout
+    when:
+      target:
+        - "*-demo"
+        - "hocs-qax"
```

Comment:
Elsewhere in the file single quotes are used, and explicit environment names are not quoted. Are these correct?

Reply:
I'll make it consistent